.travis.yml

```diff
@@ -9,3 +9,6 @@ php:
 
 before_script:
   - composer install
+  
+script:
+  - ./vendor/bin/phpunit
```

Comment:
This should be `./bin/phpunit` because of composer's `bin-dir` setting, shouldn't it?